### DISCUSSION TITLE
fabric.js responsiveness

### DISCUSSION THREAD

**Discussion by rehmangtech** [fabricjs/fabric.js#9683] [General]
Hi, I am using fabric.js to create an image edit library but its not responsive . please some can help

fabric.Image.fromURL(linkImage, (image) => { 
editor.canvas.clear(); 
setContainerHeight(image.height); 
setContainerWidth(image.width); 
editor.canvas.setWidth(containerWidth); 
editor.canvas.setHeight(containerHeight); 
editor.canvas.setZoom(1);
 editor.canvas.setBackgroundImage( image, editor.canvas.renderAll.bind(editor.canvas) ); 
 
 });

this a part of my code

i have spend a lot of time but not able to achieve it. thanks

**Comment by ShaMan123** [comment]
Provide a clear and concise **LIVE** reproduction
